02-wrangling.Rmd

```diff
@@ -256,55 +320,182 @@ lang_no_delimiter <- separate(lang_messy_longer,
 lang_no_delimiter
 ```
 
-You might notice in the table above the word `<chr>` appears beneath each of the column names. The word under the column name indicates the data type of each column. Here all of our variables are "character" data types. Recall, a character data type is a letter or a number. In the previous example, `most_at_home` and `most_at_work`  were `<dbl>` (double) (you can verify this by looking at the tables in the previous sections), which is a numeric data type. This change is due to the delimiter "/" when we read in this messy data set. R read the columns in as character types, and it stayed that way after we separated the columns. 
 
-Here it makes sense for `region`, `category`, and `language` to be stored as a character type. However, if we want to apply any functions that treat the `most_at_home` and `most_at_work` columns as a number (e.g. finding the maximum of the column), it won't be possible to do if the variable is stored as a `character`. R has a variety of data types, but here we will use the function `mutate` to convert these two columns to an "numeric" data type. `mutate` is a function that will allow us to create a new variable in our data set. We specify the data set in the first argument, and in the proceeding arguments, we specify the function we want to apply (`as.numeric`) to which columns (`most_at_home`, `most_at_work`). Then we give the mutated variable a new name. Here we are naming the columns the same names ("most_at_home", "most_at_work"), but you can call these mutated variables anything you'd like. 
+Is this data set now tidy? If we recall the three criteria for tidy data:
+
+- each row is a single observation,
+- each column is a single variable, and
+- each value is a single cell.
+
+We can see that this data now satisfies all three criteria, making it easier to analyze. However, we aren't done yet! We have a few more tasks we need to do to get this data in a more usable format, which we will see in the sections below. 
+
+### Using `select` to extract a range of columns
```

Comment:
Using three `#`'s puts this (and the `filter` and `mutate` headers) under the umbrella of tidy data, which I don't think they belong... Jenny Bryan would call these things single table manipulation functions/verbs. Let's either use something like that or put them under their own sections with two `#`'s.